### PR TITLE
Bug Fixes for welsh Yes no

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EUI-8707
https://tools.hmcts.net/jira/browse/EUI-8692
https://tools.hmcts.net/jira/browse/EUI-7677
https://tools.hmcts.net/jira/browse/EUI-8279

### Change description ###

Bug fixes for Welsh Yes no

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

File: charts/xui-webapp/values.preview.template.yaml [hunk 1]
```diff
@@ -22,6 +22,7 @@ nodejs:
      FEATURE_WORKALLOCATION_ENABLED: true
      SERVICES_CASE_JUDICIAL_API: http://rd-judicial-api-aat.service.core-compute-aat.internal
      SERVICES_LOCATION_REF_API_URL: http://rd-location-ref-api-aat.service.core-compute-aat.internal
+     SERVICES_TRANSLATION_API_URL: http://ts-translation-service-aat.service.core-compute-aat.internal
```
Comment (udaydenduluri33) [inline charts/xui-webapp/values.preview.template.yaml:25]: Can you confirm if you have added this variable to Custom-Environment-Variable.json file?

Comment (DavidJayakumar) [reply charts/xui-webapp/values.preview.template.yaml:25]: @udaydenduluri33  Yes it is available

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by Mo-Lala84 — APPROVED** [review]
approved

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Comment by DavidJayakumar** [comment]
closing this PR. All changes are already merged into https://github.com/hmcts/rpx-xui-webapp/pull/3429 which is the main branch for Welsh fixes

**Pull request closed without merging**